src/JMS/Serializer/Metadata/ClassMetadata.php

```diff
@@ -225,6 +226,7 @@ public function serialize()
             $this->accessorOrder,
             $this->customOrder,
             $this->handlerCallbacks,
+            $this->exclusionGroups,
```

Comment:
this will cause an error if will be deployed without cache cleaning, see https://github.com/schmittjoh/serializer/blob/master/src/JMS/Serializer/Metadata/PropertyMetadata.php#L169 and https://github.com/schmittjoh/serializer/blob/master/src/JMS/Serializer/Metadata/PropertyMetadata.php#L137 for a safer implementation